src/Params/EntityCreationParams.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
The concept looks very simple and straightforward. However this interface seems weak to me without more context. What helps the developer using a concrete entity creation param class? Here you only expect an array, however entities are a bit more complex. How do you see ensuring that properties, fields with all their different types are written in the right place in that return array?